README.md

```diff
@@ -14,58 +14,73 @@
 [ci-badge]: https://github.com/ngrok/ngrok-nodejs/actions/workflows/ci.yml/badge.svg
 [ci-url]: https://github.com/ngrok/ngrok-nodejs/actions/workflows/ci.yml
 
-**Note: This is beta-quality software. Interfaces may change without warning.**
+`ngrok-nodejs` is the official Node.js SDK for ngrok that requires no binaries. Quickly enable secure production-ready connectivity to your applications and services directly from your code.
 
-[ngrok](https://ngrok.com) is a globally distributed reverse proxy commonly used for quickly getting a public URL to a
-service running inside a private network, such as on your local laptop. The ngrok agent is usually
-deployed inside a private network and is used to communicate with the ngrok cloud service.
-
-This is the ngrok agent in library form, suitable for integrating directly into NodeJS
-applications. This allows you to quickly build ngrok into your application with no separate process
+[ngrok](https://ngrok.com) is a globally distributed gateway that provides secure connectivity for applications and services running in any environment.
 to manage.
 
-# Documentation
-
-A quickstart guide and a full API reference are included in the [ngrok-nodejs API documentation](https://ngrok.github.io/ngrok-nodejs/).
-
-# Quickstart
+# Installation
 
-1. Install the `ngrok-nodejs` package from
-[npm](https://www.npmjs.com/package/@ngrok/ngrok):
+Using npm:
 
 ```shell
 npm install @ngrok/ngrok
 ```
 
-2. After you've installed the package, you'll need an authtoken. Retrieve one on the
-[authtoken page of your ngrok dashboard](https://dashboard.ngrok.com/get-started/your-authtoken)
+Using yarn:
 
-3. Add the following code block using the [connect method](https://github.com/ngrok/ngrok-nodejs/blob/main/examples/ngrok-connect-minimal.js) to expose your nodejs application at port `8080` on `localhost`:
+```shell
+yarn add @ngrok/ngrok
+```
 
-```jsx
-const ngrok = require("@ngrok/ngrok");
-(async function() {
-  const listener = await ngrok.connect({ addr: 8080, authtoken_from_env: true });
-  console.log(`Ingress established at: ${listener.url()}`);
-})();
+Using pnpm:
+
+```shell
+pnpm add @ngrok/ngrok
 ```
 
-You can find more examples in [the /examples directory](https://github.com/ngrok/ngrok-nodejs/tree/main/examples).
+# Quickstart
+
+1. [Install `@ngrok/ngrok`](#installation)
+2. Export your [authtoken from the ngrok dashboard](https://dashboard.ngrok.com/get-started/your-authtoken) in your terminal
```

Comment:
Maybe mentioning `NGROK_AUTHTOKEN`

Reply:
Good call. Should have checked, was hoping this page mentioned the actual environment variable.

Reply:
The env var is down in the `Authorization` section, but I do like it here for folks that glance at the quickstart and nothing else.